### ISSUE TITLE
Add "workspace export_any" command

### ISSUE THREAD

**Issue by jric** [databricks/databricks-cli#53]
I propose to add an additional option to workspace so that the user doesn't need to know which type of object exists at the given path.  When executing a "workspace ls" it is difficult to tell which type of object it is programmatically (without spoofing a color console and parsing the color codes.)
It looks like "workspace export" will fail on a directory and "workspace export_dir" will fail on a non-directory, so neither command is a surefire way to export a given path.
I propose to add "export_any" which will call one of the two above functions as appropriate, given the path.

**Comment by jric** [comment]
Nevermind.  Reading through the code I just found the "-l" option to the "ls" command.  I now propose adding doc about this option to the README.rst.  I'll make a PR.

**Comment by jric** [comment]
Addressed with PR#54.

**Comment by jric** [comment]
Actually, I don't think this is going to solve the problem, because the output is written "space" separated, rather than tab-separated, using tabulate library.  The issue is that it will be error-prone to try to parse that, since many object-names have spaces in them.
I propose to switch to "tab" separated, or add an option to do so.

**Comment by andrewmchen** [comment]
Unfortunately it's hard to get alignment right with tab delimiters. `ls -l` will use space delimiters too. Would a new output format called --json work better? That way you can parse the json output directly with something like `jq`.

**Comment by jric** [comment]
Yes, that would be a great option.  No urgency on this idea for now....  I'm experimenting with working directly with the REST API.

**Comment by chrisst** [comment]
**This repository has been archived.**

The legacy `databricks/databricks-cli` is deprecated and has been fully replaced by the new Databricks CLI at **https://github.com/databricks/cli**.

We are closing all open issues here as part of archiving this repository. If your issue still affects the new CLI, we'd appreciate you re-opening it at https://github.com/databricks/cli/issues so the team can track and address it there.
